presto-spark-base/src/test/java/com/facebook/presto/spark/adaptive/execution/TestPrestoSparkAdaptiveJoinQueries.java

```diff
@@ -26,6 +34,34 @@ protected Session getSession()
     {
         return Session.builder(super.getSession())
                 .setSystemProperty(SPARK_ADAPTIVE_QUERY_EXECUTION_ENABLED, "true")
+                .setSystemProperty(ADAPTIVE_JOIN_SIDE_SWITCHING_ENABLED, "true")
                 .build();
     }
+
+    @Test
+    public void testQuerySucceedsWithAQE()
+    {
+        Session session = Session.builder(getSession())
+                .setSystemProperty(JOIN_REORDERING_STRATEGY, NONE.name())
+                .setSystemProperty(JOIN_DISTRIBUTION_TYPE, PARTITIONED.name())
+                .setSystemProperty(USE_HISTORY_BASED_PLAN_STATISTICS, "false")
+                .setSystemProperty(QUERY_MAX_TOTAL_MEMORY_PER_NODE, "10MB")
```

Comment:
Why do these tests have different memory limits?

Reply:
unintentional. I'll fix it.